### ISSUE TITLE
capture all keys from local machine with a lock shortcut

### ISSUE THREAD

**Issue by DavidePatria** [morrolinux/ChimeraDesk#3]
First of all, grazie Morro, the app works great and working from home has been much more painless since I do not need VNC anymore.

After using the app for a while I realised I would benefit a lot from having the following features (I'm on kde): 
- capture keys solely when the viewer window is in, and not when the viewer is in the current workspace but not in focus, but this is minor
- capture the keys without sending them to the local machine, in a fashion similar to how virt-manager does it, by requesting a command to detach them (ctrl+alt). this also applies to mouse, which moves within the viewer window unless the aforementioned shortcut is pressed. This would allow to not trigger shortcuts on the local machine in case the same shortcuts are used. In my case all the window/workspace movements have the same shortcuts on both machine and not using them is quite tedious.

These features are probably related to the spice protocol used by the viewer, but I'm not sure.

Let me know what you think.

**Comment by morrolinux** [comment]
Yes, I think the virtmanager-style detach would be really helpful, right now I'm quite busy on other projects and it's not a priority for me, but feel free to try and implement it if you wish: the code base is fairly simple and I'll gladly review your contributions if you wish to do so :)
